# Generated by Django 3.0.2 on 2020-02-19 06:45

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('errdepo_api', '0009_auto_20200202_1322'),
    ]

    operations = [
        migrations.AlterField(
            model_name='profile',
            name='image',
            field=models.BinaryField(),
        ),
        migrations.AlterField(
            model_name='report',
            name='correspondence',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='correspondenceHTML',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='description',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='descriptionHTML',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='env',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='errmsg',
            field=models.TextField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='report',
            name='fw',
            field=models.CharField(blank=True, max_length=100, null=True),
        ),
    ]
